package.json

```diff
@@ -86,5 +86,6 @@
     "@types/openseadragon": "^3.0.6",
     "@types/react-datepicker": "^4.8.0",
     "@types/validator": "^13.7.17"
-  }
+  },
+  "proxy": "https://sandbox.dissco.tech"
```

Comment:
Is this needed? And probably switch this to dev.dissco.tech

Reply:
That one slipped through, is a development variable, removed it